manifests/k/kite/kite/1.2021.610.0/kite.kite.installer.yaml

```diff
@@ -1,11 +1,18 @@
-# Created using wingetcreate
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: kite.kite
 PackageVersion: 1.2021.610.0
```

Comment:
```suggestion
PackageVersion: 1.2021.527.0
```
![image](https://user-images.githubusercontent.com/11600822/126899711-cd9fe692-aedb-4f5d-94eb-f483804c97e7.png)
Huh, when I installed it yesterday, it was `1.2021.610.0`.